src/style.css

```
@@ -1 +1,64 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  padding-right: 20px;
+}
+
+.nav__item:last-child {
+  padding-right: 0;
+}
```

use margins for distance BETWEEN elements, padding only used for distance between element's border and content
```suggestion
.nav__item {
  margin-right: 20px;
}

.nav__item:last-child {
  margin-right: 0;
}
```

---

Ooops)